src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeZone/Classes/TimeZoneSettings.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerToys.Run.Plugin.TimeZone.Properties;
+using Microsoft.PowerToys.Settings.UI.Library;
+
+namespace Microsoft.PowerToys.Run.Plugin.TimeZone.Classes
+{
+    /// <summary>
+    /// Additional settings for the time zone plugin.
+    /// </summary>
+    internal sealed class TimeZoneSettings
+    {
+        /// <summary>
+        /// Gets or sets a value indicating whether the time zone name of a time zone is shown in the results.
+        /// </summary>
+        internal bool ShowTimeZoneNames { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the time name of a time zone is shown in the results.
+        /// </summary>
+        internal bool ShowTimeNames { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the military name of a time zone is shown in the results.
+        /// </summary>
+        internal bool ShowMilitaryTimeZoneNames { get; set; }
+
+        /// <summary>
+        /// Return a list with all settings. Additional
+        /// </summary>
+        /// <returns>A list with all settings.</returns>
+        internal List<PluginAdditionalOption> GetAdditionalOptions()
+        {
+            var optionList = new List<PluginAdditionalOption>
+            {
+                new PluginAdditionalOption
+                {
+                    Key = nameof(ShowTimeZoneNames),
```

Comment:
Why do you use `nameof()` instead of simple writing a string?

(Asking because I am interested in the answer.)

Reply:
so you don't assume name of class.  this makes it more dynamic.

Reply:
> Why do you use `nameof()` instead of simple writing a string?
> 
> (Asking because I am interested in the answer.)

Nothing special, I always try to reduce use of strings.